src/BusyIcon/index.scss

```
@@ -0,0 +1,170 @@
+@use "sass:map";
+
+@use "../sass/functions";
+@use "../sass/mixins";
+@use "../sass/tokens";
+
+.busy-icon {
+  display: inline-block;
+  aspect-ratio: 1;
+  height: auto;
+
+  @include mixins.transition(background-color, short);
+
+  @each $size in tokens.$sizes {
+    &.busy-icon--#{$size} {
+      width: map.get(tokens.$icon-width, $size);
+    }
+  }
+
+  --active-color: #{map.get(tokens.$brand, white)};
+  --inactive-color: #{map.get(tokens.$brand, gray)};
+  @include mixins.light-mode {
+    --active-color: #{map.get(tokens.$brand, black)};
+    --inactive-color: #{map.get(tokens.$brand, light)};
+  }
+
+  --duration: 0.5s;
+  --initial-delay: 0.5s;
+  --start-color: transparent;
+  --step-0-color: transparent;
+  --step-1-color: transparent;
+  --step-2-color: transparent;
+  --step-3-color: transparent;
+  --end-color: transparent;
+
+  svg .loader path {
```

Remember to use BEM for CSS selectors (example here):

```suggestion
  .busy-icon__path {
```